release-notes/2.1/2.1.0-preview1.md

```diff
@@ -39,7 +39,7 @@ Microsoft.Data.SqlClient v2.1 extends support for Always Encrypted (including wi
 
 | Target Framework | Platform | New Feature support |
 |------------------|----------|----|
-| .NET Framework v4.6+ | Windows | *No changes* |
+| .NET Framework v4.6.1+ | Windows | *No changes* |
```

Comment:
I don't think we need to touch the notes for these previously released versions since they still support the net46.

Reply:
done.